### PR TITLE
Add lifecycle experimental badge

### PR DESCRIPTION
**Pull request by jdhoffa**
`experimental` or `maturing` here? 

I expect we still may want to totally revamp our `pkgdown` sites.

### CODE REVIEW

**Comment by cjyetman** [comment]
my not-so-important opinion here is that this repo is "mature" in that it does what it's supposed to do, we've been actively using it on public facing pages for many months, and there's no unsolved problems or bugs to fix (that I'm aware of). Sure, we may decide to change up the styling, but the basic functionality is pretty stable.

**Comment by AlexAxthelm** [comment]
+1 for mature.

Rationale: This repo doesn't have many PRs against it, and even if it did, none of them would be introducing what we would consider "breaking changes" for the rendered HTML.

**Comment by cjyetman** [comment]
I'd say it's "stable" then since how it's used is unlikely to change, though it's content/styling might change

**Comment by jdhoffa** [comment]
Will change tomorrow when I'm back at the computer

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 # pacta.pkgdown.template <a href="https://rmi.org"><img src="man/figures/logo.png" align="right" height="31" /></a>
 
 <!-- badges: start -->
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+[![Lifecycle: stable](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg)](https://lifecycle.r-lib.org/articles/stages.html#stable)
```
Comment (AlexAxthelm) [inline README.md:6]:
```suggestion
[![Lifecycle: stable](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg)](https://lifecycle.r-lib.org/articles/stages.html#stable)
```

Looks like you added `stable` without removing `experimental`

Comment (AlexAxthelm) [inline README.md:6]:
```suggestion
[![Lifecycle: stable](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg)](https://lifecycle.r-lib.org/articles/stages.html#stable)
```

Looks like you added `stable` without removing `experimental`

**Review by AlexAxthelm — CHANGES_REQUESTED** [review]

**Review by cjyetman — APPROVED** [review]

**Review by AlexAxthelm — APPROVED** [review]

**Pull request merged**